src/constants.min.js

```diff
@@ -0,0 +1,20 @@
rpshooter",level:0},{id:28210,name:"Stalker",class:"Sharpshooter",level:0},{id:28220,name:"Snipe",class:"Sharpshooter",level:0},{id:28157,name:"Summon Silverhawk",class:"Sharpshooter",level:0},{id:28158,name:"Summon Silverhawk",class:"Sharpshooter",level:0},{id:28160,name:"Wings of Storm",class:"Sharpshooter",level:0},{id:28170,name:"Last Rush",class:"Sharpshooter",level:0},{id:28230,name:"Fenrir's Messenger",class:"Sharpshooter",level:0},{id:28017,name:"Stand Up",class:"Sharpshooter",level:0},{id:28240,name:"Golden Eye",class:"Sharpshooter",level:0},{id:29100,name:"Meteor Stream",class:"Deadeye",level:0},{id:29060,name:"Spiral Flame",class:"Deadeye",level:0},{id:29040,name:"Sign of Apocalypse",class:"Deadeye",level:0},{id:29041,name:"Sign of Apocalypse",class:"Deadeye",level:0},{id:29020,name:"Enforce Execution",class:"Deadeye",level:0},{id:29090,name:"Death Fire",class:"Deadeye",level:0},{id:29070,name:"Triple Explosion",class:"Deadeye",level:0},{id:29071,name:"Triple Explosion",class:"Deadeye",level:0},{id:29072,name:"Triple Explosion",class:"Deadeye",level:0},{id:29073,name:"Triple Explosion",class:"Deadeye",level:0},{id:29120,name:"AT02 Grenade",class:"Deadeye",level:0},{id:29180,name:"Spiral Tracker",class:"Deadeye",level:0},{id:29140,name:"Plasma Bullet",class:"Deadeye",level:0},{id:29080,name:"Catastrophe",class:"Deadeye",level:0},{id:29150,name:"Tumbling",class:"Deadeye",level:0},{id:29170,name:"Shotgun Rapid Fire",class:"Deadeye",level:0},{id:29200,name:"Dexterous Shot",class:"Deadeye",level:0},{id:29190,name:"Cruel Tracker",class:"Deadeye",level:0},{id:29155,name:"Stand Up",class:"Deadeye",level:0},{id:29210,name:"Somersault Shot",class:"Deadeye",level:0},{id:29220,name:"Equilibrium",class:"Deadeye",level:0},{id:29230,name:"Last Request",class:"Deadeye",level:0},{id:29240,name:"Aimed Shot",class:"Deadeye",level:0},{id:29250,name:"Clay Bombardment",class:"Deadeye",level:0},{id:29260,name:"Quick Shot",class:"Deadeye",level:0},{id:29261,name:"Quick Shot",class:"Deadeye",level:0},{id:29262,name:"Quick Shot",class:"Deadeye",level:0},{id:29110,name:"Shotgun Dominator",class:"Deadeye",level:0},{id:29270,name:"Perfect Shot",class:"Deadeye",level:0},{id:29280,name:"Bursting Flare",class:"Deadeye",level:0},{id:29160,name:"Change Stance",class:"Deadeye",level:0},{id:29161,name:"Change Stance",class:"Deadeye",level:0},{id:29162,name:"Change Stance",class:"Deadeye",level:0},{id:29163,name:"Change Stance",class:"Deadeye",level:0},{id:29164,name:"Change Stance",class:"Deadeye",level:0},{id:29050,name:"Hour of Judgment",class:"Deadeye",level:0},{id:30020,name:"Enter Barrage Mode",class:"Artillerist",level:0},{id:30021,name:"Exit Barrage Mode",class:"Artillerist",level:0},{id:30050,name:"Enhanced Shell",class:"Artillerist",level:0},{id:30240,name:"Barrage: Basic Attack",class:"Artillerist",level:0},{id:30060,name:"Freeze Shell",class:"Artillerist",level:0},{id:30070,name:"Buckshot",class:"Artillerist",level:0},{id:30080,name:"Multiple Rocket Launcher",
```

Comment:
It includes additional translations/skills. It's minified because my editor auto minifies large JSON and I had this as a JSON import for testing at some point.

Unminified in commit. If you don't want the extra information lmk and I'll revert it to your version.